lib/routeUtils.ts

```diff
@@ -20,11 +20,3 @@ export const getPageNameUrl = (): string => {
 export const isSplashPage = (): boolean => {
   return getPageNameUrl() === "splash";
 };
-
```

Comment:
Just to make sure, this function is being removed because it is not being used in the code base?